DiagnosticsExtension/Models/ConnectionStringValidator/ServiceBusValidator.cs

```diff
@@ -20,10 +20,9 @@ namespace DiagnosticsExtension.Models.ConnectionStringValidator
     public class ServiceBusValidator : IConnectionStringValidator
     {
         public string ProviderName => "Microsoft.Azure.ServiceBus";
-
+        public ConnectionStringValidationResult response = null;
```

Comment:
See my comment in blob validator on this

Reply:
fixed